tests/conftest.py

```
@@ -312,7 +315,24 @@ def mock_memory_server():
+            with httpx.Client(timeout=timeout_seconds, proxy=None, trust_env=False) as client:
+                response = client.get("http://127.0.0.1:48912/new_dialog/healthcheck")
+            return response.status_code == 200
+        except (httpx.HTTPError, OSError):
+            return False
+
+    try:
+        if _is_memory_server_ready():
+            yield
+            return
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)